CHANGES.rst

```diff
@@ -2,6 +2,21 @@
 Changes
 =======
 
+0.17.3 (2024-06-NN)
+===================
+
+-   Replaced a use of the deprecated pkg_resources_ module with importlib_.
+
+    .. _pkg_resources: https://setuptools.pypa.io/en/latest/pkg_resources.html
+    .. _importlib: https://docs.python.org/3/library/importlib.html
+
+-   Added support for centralized request fingerprints on Scrapy 2.7 and
+    higher.
```

Comment:
It seems this change was a part of 0.17.2 release - what's the reason it's included here?

Reply:
A malfunctioning brain :facepalm: